fvp/spmc_manifest.dts

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
You might want to keep an eye on this PR: https://review.trustedfirmware.org/c/TF-A/trusted-firmware-a/+/15505
It enables Measured Boot with Hafnium.